.build/parent-pom-template.xml

```diff
@@ -747,14 +813,20 @@
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-transport-native-epoll</artifactId>
-        <version>4.1.94.Final</version>
+        <version>4.1.96.Final</version>
```

Comment:
can you define netty version in a property? (no need to rerun the tests)

Reply:
@jacek-lewandowski we will dedicate a separate ticket for this, I just do not want to have it inconsistent, all versions should go to properties IMHO.

Reply:
ok